packages/internals/src/engine-commands/getDmmf.ts

```diff
@@ -61,8 +62,6 @@ type GetDmmfErrorInit = {
 
 export class GetDmmfError extends Error {
   constructor(params: GetDmmfErrorInit) {
-    const headline = chalk.redBright.bold('Get DMMF: ')
-
```

Comment:
I don't remember that we decided to remove this part? 🤔

Reply:
https://github.com/prisma/prisma/pull/14512/files#r954200035